glide.lock

```diff
@@ -129,7 +130,7 @@ imports:
   subpackages:
   - prometheus
 - name: github.com/prometheus/client_model
-  version: fa8ad6fec33561be4280a8f0514318c79d7f6cb6
+  version: 6f3806018612930941127f2a7c6c453ba2c527d2
```

Comment:
actually since this is only used in etcd package, we should probably not pin it, so that etcd can pin it to a version if needed. If we pin it now, we might forgot why we pinned it in the first place and hard to unpin in the future.